### PR TITLE
Docs: add code_of_conduct in project

### PR DESCRIPTION
**Pull request by victorevh**
## 📙 Description
We received feedback regarding the absence of a Code of Conduct in our project repository. This is essential to ensure a welcoming and inclusive community environment where contributors feel comfortable making contributions.

This change ensures the addition of a comprehensive Code of Conduct to the project, outlining the expected behaviors and the consequences for not adhering to them. This update aims to make all community members feel respected and valued, promoting a positive and inclusive community space.

In the evaluation, it was also mentioned that the README.md file should include a section informing and directing contributors to the Code of Conduct for more details.

## 🎯 Actions
Addition of the CODE_OF_CONDUCT.md file to the project repository, outlining the expected behavior, responsibilities of the project maintainers, and the enforcement protocols for unacceptable behavior.
Inclusion of a section in the README.md file to inform and guide contributors to the Code of Conduct. This section provides a brief overview and a link to the full Code of Conduct for further details.
Ensuring all community and project channels are aware of the existence and the importance of the new Code of Conduct.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Scope
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team by using `#moderation` in the Discord community. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality concerning the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (rubemfsv) [inline CODE_OF_CONDUCT.md]: Change this discord part to “by emailing rfsv@cesar.school” for now, I think it’s better once we don’t have a discord channel. I’m open to suggestions in the text, this is only an example

File: README.md [hunk 2]
```diff
@@ -181,3 +181,6 @@ Meet the talented individuals who have contributed to Clean React App project:
   <img src="https://contrib.rocks/image?repo=rubemfsv/clean-react-app" />
 </a>
 
+## 📜 Code of Conduct
```
Comment (rubemfsv) [inline README.md]: Put it after the issues section, don’t forget to use the hr br tags according to the readme pattern

**Review by rubemfsv — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (victorevh) [reply CODE_OF_CONDUCT.md]: Thanks @rubemfsv! I Fix it

File: README.md (hunk 2 above)
Comment (victorevh) [reply README.md]: Tks @rubemfsv!! I change this!

**Review by rubemfsv — APPROVED** [review]

**Pull request merged**